autoload/lsc/diff.vim

```diff
@@ -27,21 +27,21 @@ function! lsc#diff#compute(old, new) abort
   return result
 endfunction
 
-if has('lua') && !exists('s:lua')
+if (has('lua') || has('nvim')) && !exists('s:lua')
```

Comment:
I would instead use `has('nvim-0.4.0')` a bit older version of neovim doesn't support lua. I can't remember the exact version when neovim started to ship lua.

I also prefer to save it in `s:has_lua` since this check will happen in multiple places.

Reply:
It looks like the `nvim_eval` function goes back to at least `v0.2.0`, but that version doesn't compile for me so I can't check whether there are any behavior differences that would break this. It's an optimization I'll go ahead and use `0.4.0` as you suggest since it's safe.